src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/repository/InterventionFilterRepositoryImplTest.kt

```diff
@@ -237,4 +238,18 @@ class InterventionFilterRepositoryImplTest @Autowired constructor(
     assertThat(found.size).isEqualTo(4)
     assertThat(found).contains(futureIntervention)
   }
+
+  @Test
+  fun `get interventions with an ended contract in the past and one in the future`() {
```

Comment:
I like this assertion at the end.
```
    assertThat(found).doesNotContain(endedIntervention)
    assertThat(found).contains(futureEndIntervention)
```

Could this be reflected in the test name, with assertive language (e.g. "filters out", "removes"), please?

Reply:
updated.